crates/ruma-common/src/time.rs

```diff
@@ -21,9 +21,13 @@ impl MilliSecondsSinceUnixEpoch {
         Some(Self(millis))
     }
 
-    /// The current milliseconds since the unix epoch.
+    /// The current system time in milliseconds since the unix epoch.
     pub fn now() -> Self {
-        Self::from_system_time(SystemTime::now()).unwrap()
+        #[cfg(not(all(target_arch = "wasm32", target_os = "unknown", feature = "js")))]
+        return Self::from_system_time(SystemTime::now()).unwrap();
```

Comment:
Not sure what the policy on unwraps in Ruma is, but I prefer to use expects unless it's test code.

Reply:
Right, why not.